Dart/src/com/jetbrains/lang/dart/ide/formatter/DartBlock.java

```diff
@@ -3,14 +3,19 @@
 import com.intellij.formatting.*;
 import com.intellij.formatting.templateLanguages.BlockWithParent;
 import com.intellij.lang.ASTNode;
+import com.intellij.openapi.util.Key;
+import com.intellij.psi.PsiElement;
```

Comment:
A bunch of unused imports. Before commit please make sure that your code doesn't add new warnings highlighted by IDE or warnings are justified.